src/common/CardSection.js

```diff
@@ -39,6 +39,26 @@ const CardSection = ({ to, title, image, className, children, external }) => {
     )
   }
 
+  if (external && external.startsWith('http')) {
```

Comment:
this if for "external" is duplicated. You should remove it.

Reply:
I haven't look at the whole source code yet. But maybe the intention could be check if external is `undefined` before invoking a method on it?

On a side note, modern Javascript supports nullish coalescing operator, so it can be written as `external?.startsWith('http') === true` as well.

Reply:
I was referring that the code block starting with `if (external && external.startsWith('http')) {` is duplicated just above. Then it is better to remove it.

Reply:
Oh, I totally missed that. Sorry, I should have see the whole source before writing a comment 😅 

Anyway, I agree that it's redundant and thanks for the review!

Reply:
> this if for "external" is duplicated. You should remove it.

?!
Not sure how that duplicate was created -.-
Must have accidentally spammed CTRL+V too much without noticing haha -.-

Does running Lint notice these errors? I haven't tried it yet, but perhaps I should.